JavaScripit/index.js

```diff
@@ -18,7 +18,74 @@ var skillsList = skillsSection.querySelector('ul');
 for (var i = 0; i < skills.length; i++) {
     var skill = document.createElement("li");
     skill.innerText = skills[i];
-    skillsList.appendChild(skill);
+    skillsList.appendChild(skill); 
+}
+// DOM Selection
+const messageForm = document.forms.leave_message;
+// Event Listener
+messageForm.addEventListener('submit', function (event) {
+    event.preventDefault(); // Prevents the default refreshing behavior of the "submit" event
+
+    // Retrieve form field values
+    const usersName = event.target.usersName.value;
+    const usersEmail = event.target.usersEmail.value;
+    const usersMessage = event.target.usersMessage.value;
```

Comment:
When you have repetitive assignments like this, [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring) may be useful.

Reply:
It's a new concept definitely will study it in deep about destructuring assignments.